firefox_background.js

```diff
@@ -0,0 +1,13 @@
+ChromeService.newTabUrl = "about:newtab";
```

Comment:
Please use the existing file `firefox_bg.js`.

Reply:
Please refer to https://github.com/brookhong/Surfingkeys/blob/d208f073d175276830d698cc071d29c4a5e6920d/gulpfile.js#L113

I couldnt use the existing firefox_bg.js because it is sourced before
background.js (which defines functions then used in background.js) but this
depends on variables defined in background.js. Therefore it must be sourced
after.

Maybe changing the file name like firefox_post_bg.js would be more
descriptive.

Reply:
Then you could follow the pattern of `_applyProxySettings` to define an empty stub of `getContainerName` for `Chrome`, and move those from `firefox_background.js` into `background.js`, which could also avoid export the private `_response`.